udf/bits_to_last_seen_int.sql

```diff
@@ -0,0 +1,41 @@
+/*
+Given a BYTE, return the number of days since the client was
+last seen.
+
+If no bits are set, returns NULL, indicating we don't know.
+Otherwise the results are 0-indexed, meaning \x01 will return 0.
+
+Tests showed this being 5-10x faster than the simpler alternative:
+CREATE TEMP FUNCTION
+  udf_bits_to_last_seen_int(b BYTES) AS ((
+    SELECT MIN(n)
+    FROM UNNEST(GENERATE_ARRAY(0, 364)) AS n
+    WHERE BIT_COUNT(SUBSTR(b >> n, -1) & b'\x01') > 0));
```

Comment:
I really appreciate the documentation here of alternate approaches and reasoning behind the implementation.